### PR TITLE
File id encoding

### PR DESCRIPTION
**Pull request by kastoestoramadus**
FYI: fileId encoding is probably not used by Frontend yet
+ rejecting file upload requests without file length provided

### CODE REVIEW

**Comment by mcarolan** [comment]
does test coverage need to be introduced (still looking at the wider context sorry)?

**Comment by kastoestoramadus** [comment]
@mcarolan 
encoding is not used by this module. it is tested on the backend and it is tested there.
There are some code that is shared and needs to be in synch between backend and frontend.
About the no file lenght I am still thinking.

File: app/uk/gov/hmrc/fileupload/controllers/FileUploadController.scala [hunk 1]
```diff
@@ -65,15 +65,17 @@ class FileUploadController( redirectionFeature: RedirectionFeature,
...
-              uploadToQuarantine(key, file.ref.inputStream, file.ref.size).flatMap { uploadResult =>
-                val fileRefId = FileRefId(uploadResult.getVersionId)
-                commandHandler.notify(QuarantineFile(envelopeId, fileId, fileRefId, created = now(), name = file.filename,
-                  contentType = file.contentType.getOrElse(""), file.ref.size, metadata = metadataAsJson(formData)))
-                  .map {
-                    case Xor.Right(_) => Ok
-                    case Xor.Left(e) => Status(e.statusCode)(e.reason)
-                  }
-              }
+              if(file.ref.size != null) {
```
Comment (mcarolan) [inline app/uk/gov/hmrc/fileupload/controllers/FileUploadController.scala]: lift to `Option` and then `fold` could be more idiomatic than comparison to `null`? not a big deal though

**Comment by kastoestoramadus** [comment]
@mcarolan Now there is no new feature to have new tests on this side of the service.

File: app/uk/gov/hmrc/fileupload/controllers/FileUploadController.scala (hunk 1 above)
Comment (kastoestoramadus) [inline app/uk/gov/hmrc/fileupload/controllers/FileUploadController.scala]: Check the new version.

**Pull request merged**